include/zephyr/senss/senss_sensor_types.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SENSS_SENSOR_TYPES_H_
+#define SENSS_SENSOR_TYPES_H_
+
+/**
+ * @brief Sensor Types Definition 
+ *
+ * Sensor types definition followed HID standard. 
+ * https://usb.org/sites/default/files/hutrr39b_0.pdf
+ *
+ * TODO: will add more types
+ *
+ * @addtogroup senss_sensor_types
+ * @{
+ */
+
+/**
+ * sensor category light
+ */
+#define SENSS_SENSOR_TYPE_LIGHT_AMBIENTLIGHT                   0x41
+#define SENSS_SENSOR_TYPE_LIGHT_CONSUMER_INFRARED              0x42
+
+/**
+ * sensor category motion 
+ */
+#define SENSS_SENSOR_TYPE_MOTION_ACCELEROMETER_1D              0x71

```

Comment:
We should have space to define new sensors - virtual or not, but it does not make sense to define sensor that are not implemented currently by the susbsystem.